backend/pyproject.toml

```diff
@@ -12,12 +12,14 @@ pydantic = "^1.10.7"
 websocket = "^0.2.1"
 cached-property = "^1.5.2"
 uvicorn = "^0.22.0"
+websockets = "^11.0.3"
 
 
 [tool.poetry.group.dev.dependencies]
 ruff = "^0.0.264"
 pytest = "^7.3.1"
 ipdb = "^0.13.13"
+websocket-client = "^1.5.2"
```

Comment:
huh? no unit tests and yet client is present? weird, do you use it in code?

Reply:
:((((((((

Reply:
I did manual testing with it- we need to add an issue for tests